go/test/endtoend/recovery/pitr/shardedpitr_test.go

```diff
@@ -448,8 +448,9 @@ func initializeCluster(t *testing.T) {
 
 	for _, shard := range clusterInstance.Keyspaces[0].Shards {
 		for _, tablet := range shard.Vttablets {
+
 			for _, query := range queryCmds {
-				_, err = tablet.VttabletProcess.QueryTablet(query, keyspace.Name, false)
+				_, err = tablet.VttabletProcess.QueryTabletWithSuperReadOnlyHandling(query, keyspace.Name, false)
```

Comment:
Maybe we can get rid of the entire queryCmds list from here. The commands that are being run can be added to the init_db.sql file. So we can drop the `create database` command and put all the others in the init_db.sql file. We already have code that does something similar in a different test - `LaunchCluster` in backup_utils.go

```
firstPart := splitString[0] + cluster.GetPasswordUpdateSQL(localCluster)
	sql = firstPart + splitString[1]
	newInitDBFile = path.Join(localCluster.TmpDirectory, "init_db_with_passwords.sql")
	err = os.WriteFile(newInitDBFile, []byte(sql), 0666)
```